experimental/PropertyDDS/packages/property-inspector-table/src/InspectorTableTypes.ts

```diff
@@ -187,13 +266,13 @@ export interface IExpandedMap {
   [key: string]: boolean;
 }
 
-export interface IInspectorTableState {
+export interface IInspectorTableState<T = any> {
   childToParentMap: { [key: string]: string; };
-  commitHistoryVisible: boolean;
+  commitHistoryVisible?: boolean;
   currentResult?: number;
   foundMatches: IInspectorSearchMatch[];
   matchesMap: IInspectorSearchMatchMap;
-  editReferenceRowData: IInspectorRow | null;
+  editReferenceRowData: T | null;
```

Comment:
I get eslint warnings for such cases
> Usage of "null" is deprecated except when describing legacy APIs; use "undefined" instead `eslint` [@rushstack/no-new-null](https://www.npmjs.com/package/%40rushstack/eslint-plugin)

Their arguments in description seem reasonable.

Reply:
Yes, I tried to replace them, it broke the table, it seems in some cases its distinguished from `undefined` case. It will require a careful change, I will try to get rid of these nulls in the following PR.